src/rmc-compiler/build.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use std::env;
+use std::path::PathBuf;
+use std::process::Command;
+
+macro_rules! path_str {
+    ($input:expr) => {
+        &$input.iter().collect::<PathBuf>().to_string_lossy().into_owned()
```

Comment:
Did we want to do a lossy conversion, or to fail fast here?

Reply:
Sure, I can fall fast.